docs/developer-guide.md

```diff
@@ -229,3 +229,21 @@ Convention | Use case | Example
 `{XXX}Map`, `{XXX}Builder` | Builds an **immutable** `{XXX}Map` from a **mutable** `{XXX}Builder` | `DependencyMap`, `DependencyMapBuilder`
 `{XXX}Provider`   | Groups **instance** helper methods for **Get** or **Resolve** | `MonikerProvider`
 `{XXX}Utility`    | Groups **static** helper methods | `GitUtility`
+
+
+## Regression Test Expected Diffs
+
+- Sometimes the contributors list may change.
+    - There will be a new contributor when a new contributor edits the article.
```

Comment:
`You may check against corresponding GitHub page to double confirm whether the change is expected or not.`